content/en/docs/concepts/storage/volume-attributes-classes.md

```diff
@@ -0,0 +1,119 @@
+---
+reviewers:
+- msau42
+- xing-yang
+title: Volume Attributes Classes
+content_type: concept
+weight: 40
+---
+<!-- overview -->
+
+This page assumes that you are familiar with [StorageClasses](/docs/concepts/storage/storage-classes/),
+[volumes](/docs/concepts/storage/volumes/) and [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
+in Kubernetes.
+
+<!-- body -->
+
+## Introduction
+
+A VolumeAttributesClass provides a way for administrators to describe the mutable
+"classes" of storage they offer. Different classes might map to different quality-of-service levels.
+Kubernetes itself is unopinionated about what these classes represent.
+
+
+## The VolumeAttributesClass Resource
+
```

Comment:
Do you need to enable any feature gates and API groups to use this API?

Reply:
Yes feature gates, it is captured in the blog post.

Reply:
Cool. Please cover the details in the documentation as well.

Reply:
Which session I should add this? Should it be in introduction?

Reply:
I don't see that feature gate on the feature gates page for the `dev-1.29` branch
https://github.com/kubernetes/website/blob/dev-1.29/content/en/docs/reference/command-line-tools-reference/feature-gates.md

If it's an alpha feature gate then it should be added

Reply:
Updated the feature-gates.md